.travis/script.sh

```diff
@@ -62,6 +62,7 @@ wait_for_pulp 20
 # Run functional tests
 pytest -v -r sx --color=yes --pyargs pulpcore.tests.functional || show_logs_and_return_non_zero
 pytest -v -r sx --color=yes --pyargs pulp_file.tests.functional || show_logs_and_return_non_zero
+pytest -v -r sx --color=yes --pyargs pulp_certguard.tests.functional || show_logs_and_return_non_zero
```

Comment:
What if `tests` are not included to the package? as you cloned I recommend using filsystem lookup instead of module lookup:

```suggestion
pytest -v -r sx --color=yes pulp-certguard/tests/functional || show_logs_and_return_non_zero
```